<launch>
    <!-- Load parameters from config.yaml -->
    <rosparam file="$(find openpcdet)/launch/config.yaml" command="load" ns="object_3d_detector_node"/>
    
    <node pkg="openpcdet" type="inference.py" name="object_3d_detector_node" output="screen"/>

    <!-- Load Parameter -->
  
    <param name="use_cpu"           value="true" />

    <!-- Start yolov5 and ros wrapper -->

    <!-- <node pkg="yolov8_ros" type="yolo_v8.py" name="yolov8_ros" output="screen" >
    <param name="weight_path"       value="$(find yolov8_ros)/weights/yolov8s.pt"/>
    <param name="image_topic"       value="/lucid_image0" />
    <param name="pub_topic"         value="/yolov8/BoundingBoxes" />
    <param name="camera_frame"       value="camera_color_frame"/>
    <param name="visualize"       value="false"/>
    <param name="conf"              value="0.3" />
    </node> -->
</launch>